run_acceptance_tests.sh

```diff
@@ -1,11 +1,30 @@
 #!/bin/bash -e
 ENV=${1:-local}
-BROWSER=${2:-chrome}
+BROWSER=${2:-remote-chrome}
+
+# sm ports needed when running assets-frontend locally via service manager
+sm_port_mapping=$(sm -s | grep PASS | awk '{ print $12"->"$12 }' | paste -sd "," -)
+
+# port mapping for running contact-frontend service
+service_port_mapping="6001->6001,"
+port_mappings="$service_port_mapping$sm_port_mapping"
+
+IMAGE=artefacts.tax.service.gov.uk/chrome-with-rinetd:latest
+
+docker pull ${IMAGE} \
```

Comment:
Very minor but there is [a comment](https://github.com/hmrc/platform-example-ui-journey-tests/blob/974bf808f9e95aef9c803f7e837b9c91bd814c15/run_browser_with_docker.sh#L55) in the original script that I personally find useful as I work on Linux. Might be worth adding that to avoid any rooting around later.

Reply:
@tmc-ee Added as per your suggestion.